gradle.properties

```diff
@@ -1,11 +1,11 @@
 org.gradle.jvmargs=-Xmx2G
 
 # Fabric Properties from https://fabricmc.net/develop
-minecraft_version=1.19
-yarn_mappings=1.19+build.4
+minecraft_version=1.19.1
+yarn_mappings=1.19.1+build.3
```

Comment:
Latest version is `1.19.1+build.5`

Reply:
https://github.com/RaptaG/StairAutoJump/commit/93e5de93ff888105ab5b26b7c7fa9bea068b346a